server/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -595,10 +595,7 @@ private ExternalReaderManager createReaderManager() throws EngineException {
             try {
                 final ElasticsearchDirectoryReader directoryReader =
                     ElasticsearchDirectoryReader.wrap(DirectoryReader.open(indexWriter), shardId);
-                internalReaderManager = new ElasticsearchReaderManager(
-                    directoryReader,
-                    new RamAccountingRefreshListener(engineConfig.getCircuitBreakerService())
-                );
+                internalReaderManager = new ElasticsearchReaderManager(directoryReader, (x, y) -> {});
```

Comment:
Looks like the `refreshListener` could be removed from `ElasticsearchReaderManager` ?